C++/C.sublime-syntax

```diff
@@ -1041,8 +1041,8 @@ contexts:
       captures:
         1: meta.preprocessor.c keyword.control.import.c
       pop: true
-    - match: (?=^\s*#\s*(elif|else)\b)
-      push: preprocessor-elif-else-branch-global
+    - match: (?=^\s*#\s*(elif|elifdef|elifndef|else)\b)
+      push: preprocessor-elif-elifdef-elifndef-else-branch-global
```

Comment:
We probably don't need to rename contexts. Adding keywords to patterns is sufficient.

Reply:
Reverted in latest revision.